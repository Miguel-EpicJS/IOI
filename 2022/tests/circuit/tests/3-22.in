385045b28fbc1903eca997d513040235e0229e01
44 947 5
-1 0 1 1 2 1 1 0 1 3 9 1 9 7 5 2 15 10 16 2 14 2 13 15 3 3 11 26 25 27 26 27 28 32 28 29 33 28 25 38 32 39 37 42 0 0 0 1 0 1 1 1 1 1 1 1 2 0 2 2 2 2 0 0 2 1 3 2 1 2 3 3 2 3 2 2 1 3 4 4 2 4 2 1 4 4 4 2 3 2 4 5 0 0 4 2 2 6 4 5 4 1 2 7 1 7 2 3 6 4 7 3 5 3 2 3 1 4 6 3 6 7 4 1 1 0 2 2 3 4 4 2 5 7 8 4 5 9 9 2 8 2 8 4 1 8 6 7 4 6 1 9 7 8 9 9 7 11 7 8 1 9 11 9 8 11 7 11 9 6 3 9 3 8 8 3 9 6 3 7 3 9 3 9 0 4 7 9 7 3 2 2 5 12 12 6 4 4 9 3 6 12 12 8 12 13 4 6 11 8 7 8 13 3 9 13 9 13 13 11 11 8 5 13 3 13 9 7 13 2 3 1 14 6 8 13 15 7 2 7 11 6 12 9 13 13 6 7 11 7 10 6 5 15 11 15 13 10 1 9 12 3 15 13 11 9 0 5 11 13 13 13 13 13 13 5 14 13 15 19 2 3 11 3 11 11 14 2 6 3 4 3 9 13 16 4 15 20 16 11 4 15 11 15 13 16 20 6 8 19 0 6 20 11 20 6 6 11 3 20 20 4 20 20 20 17 18 11 11 18 13 4 18 20 15 19 20 19 9 15 20 21 7 4 19 8 20 15 9 20 14 21 1 11 15 21 4 20 11 16 9 13 9 21 21 19 23 21 1 19 7 20 21 9 12 21 3 21 15 8 21 11 1 6 24 21 15 13 21 1 13 21 11 15 9 21 21 23 24 23 9 22 12 21 14 23 18 21 15 21 23 21 5 16 6 16 10 21 23 23 21 19 2 1 9 21 19 4 20 23 13 23 24 13 13 4 23 24 24 20 26 26 26 2 26 19 6 26 15 24 2 18 20 26 16 20 27 23 9 24 7 15 24 25 26 23 25 15 23 19 15 13 27 21 21 27 26 28 25 27 20 13 28 9 24 21 3 27 27 3 15 26 13 24 28 28 21 15 28 26 26 7 2 25 7 20 19 28 28 27 24 4 28 23 13 29 20 15 15 13 24 24 27 7 24 28 8 1 26 10 13 28 24 29 23 29 25 27 25 20 27 27 15 28 9 26 21 28 29 29 20 0 28 26 24 14 25 29 32 19 28 30 2 29 28 23 32 26 21 27 27 24 25 9 6 27 29 28 20 24 9 21 27 19 32 14 29 9 32 15 21 29 27 21 26 15 23 30 25 14 32 28 33 20 28 21 24 29 31 28 27 29 30 4 26 22 28 29 32 24 9 28 6 15 27 30 26 33 15 32 33 20 28 34 7 28 25 13 31 30 34 29 28 9 31 26 28 33 21 30 21 23 26 29 11 31 28 27 33 4 23 18 27 33 33 31 2 32 28 26 25 31 23 27 32 21 24 26 32 23 12 21 27 21 6 12 27 34 27 3 2 26 33 26 27 13 31 28 31 15 36 36 32 27 26 36 36 35 28 29 36 31 36 35 36 28 19 36 13 33 20 28 35 36 36 35 18 35 36 35 4 36 23 29 36 36 35 21 31 35 36 36 33 35 20 30 29 29 36 31 36 37 35 37 13 32 21 36 35 36 37 37 31 37 36 30 11 37 37 36 37 35 35 32 38 38 28 32 38 37 36 37 36 37 36 36 37 19 36 23 26 26 28 36 37 37 38 36 30 36 35 37 13 27 38 37 36 32 28 34 38 36 37 38 36 33 20 34 37 39 35 26 37 33 37 37 39 31 39 38 36 35 29 35 37 24 38 38 39 39 38 40 19 35 36 38 21 37 32 38 40 36 36 38 38 41 41 41 39 41 41 26 39 42 42 39 38 42 38 42 40 40 42 41 41 41 41 42 29 42 38 37 34 24 42 42 41 41 39 41 41 42 36 41 42 41 41 41 42 42 42 42 24 39 37 38 41 42 39 41 42 39 42 41 41 42 41 36 39 42 27 39 36 42 42 42 42 42 32 33 25 37 43 42 39 42 38 42 38 39 41 35 36 25 42 41 39 35 38 42 36 42 26 15 43 37 39 36 30 42 39 41 42 41 41 41 38 42 41 41 36 43 42 41 41 41 42 37 38 41 42 42 42 41 41 41 35 42 42 42 41
1 1 0 0 0 0 1 0 1 0 1 1 0 0 0 0 1 0 0 0 0 1 1 0 1 0 0 1 1 1 0 0 1 0 0 1 0 0 0 1 0 1 1 0 1 0 0 1 1 1 0 0 0 0 0 1 0 0 0 0 1 0 1 0 0 1 0 1 1 1 0 0 1 0 1 1 0 0 1 1 0 0 0 0 0 0 0 1 1 1 0 1 1 1 1 0 0 1 1 0 1 0 1 1 0 1 1 1 0 1 1 1 0 0 0 1 0 0 1 1 0 0 1 1 1 0 0 0 0 0 0 1 0 0 0 0 1 0 1 1 0 0 0 0 0 0 1 0 0 0 1 0 1 0 0 0 1 1 0 0 0 0 1 0 0 0 1 1 0 0 1 0 0 0 0 1 0 1 1 1 1 0 0 1 0 0 1 0 0 1 1 0 0 1 0 0 0 0 0 0 1 0 0 1 1 1 0 0 0 0 1 0 0 0 1 1 1 1 0 0 0 0 1 1 0 1 0 0 0 1 0 0 0 0 0 1 1 1 1 1 1 0 0 0 1 1 1 1 0 0 1 1 1 1 0 1 1 0 1 1 1 0 1 1 0 1 1 1 1 0 0 0 0 0 0 0 1 1 0 1 1 0 1 0 1 1 1 0 1 1 1 1 0 0 1 0 1 1 1 0 0 0 0 0 1 1 0 0 1 0 0 1 0 1 0 1 0 1 1 0 0 1 1 0 1 1 1 0 1 1 0 0 1 1 0 0 0 0 1 0 0 0 0 0 1 0 1 0 0 0 0 1 1 0 0 1 0 1 0 1 0 0 0 1 0 0 0 1 1 1 1 1 1 0 0 1 0 0 1 0 0 1 0 0 0 1 0 0 0 0 0 1 0 0 1 0 0 0 1 0 1 1 0 1 1 0 0 1 0 0 1 0 1 0 1 1 1 0 1 0 1 1 1 1 0 0 1 0 1 0 0 0 0 0 1 0 1 1 0 1 1 0 1 1 1 1 1 0 1 0 0 0 1 0 0 1 0 1 0 0 1 1 1 0 1 0 0 1 0 0 1 0 1 0 0 0 1 1 1 0 0 0 1 0 0 1 0 0 0 1 1 1 0 1 1 1 0 0 1 0 0 0 1 0 1 0 0 1 0 1 1 0 1 0 1 0 0 1 1 1 0 0 0 0 0 0 1 1 0 1 0 1 1 0 0 0 0 1 0 0 1 0 1 0 1 0 0 0 0 0 1 1 1 1 0 1 0 0 0 0 0 0 1 0 1 0 1 0 0 0 0 1 0 0 0 1 0 0 0 1 0 0 1 0 1 1 1 1 0 0 0 0 1 0 1 0 1 0 1 0 0 0 1 0 0 1 1 1 1 1 1 0 0 0 0 1 1 1 1 1 0 1 0 1 1 1 1 1 0 1 1 0 0 1 1 1 1 1 1 0 1 1 1 0 1 1 1 0 0 0 1 1 0 0 1 0 0 0 0 0 0 1 1 0 0 0 0 0 0 0 1 1 0 1 1 1 1 0 0 1 1 1 0 1 0 1 1 1 1 1 1 0 1 0 1 0 1 0 0 0 1 0 1 1 1 1 0 0 0 0 1 0 0 1 1 0 1 1 1 1 1 0 0 1 0 0 1 0 1 1 0 0 1 0 1 0 1 0 1 1 0 0 1 0 1 0 1 1 0 1 1 1 1 1 1 0 1 0 1 1 1 0 1 1 1 0 0 1 0 0 0 1 1 0 0 0 1 1 0 0 0 1 1 0 0 0 1 0 0 1 0 1 0 1 0 0 1 1 0 1 1 0 0 0 0 1 1 0 0 0 1 0 0 1 1 1 1 0 1 0 0 1 0 0 0 0 0 0 0 0 0 1 1 1 1 0 1 0 1 0 0 1 0 0 0 1 0 1 1 1 1 1 1 0 0 0 0 0 0 1 0 1 1 1 1 1 1 0 0 1 1 0 1 1 1 1 1 1 1 0 0 1 1 0 1 0 0 1 1 1 0 1 0 1 1 1 1 0 1 0 0 1 0 0 0 0 0 1 0 1 0 0 1 0 0 0 1 0 0 0 1 0 1 1 1 0 0 1 1 1 1 1 0 1 0 0 1 1 0 1 0 0 0 1 1 0 0
170 751
146 916
144 875
355 417
366 672
