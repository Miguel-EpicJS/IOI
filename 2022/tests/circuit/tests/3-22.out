4faa78c22ee35083e49b65601698a1bd98a87a0f
OK
319941708
871406470
716326204
656940502
100714900
